### ISSUE TITLE
Notifications: "participation ends" email is sent twice to participants who follow a project & Moderators

### ISSUE THREAD

**Issue by lthueer** [liqd/adhocracy-plus#1663]
**URL:** https://aplus-dev.liqd.net/teststadt-luca/projects/test-notifications/
**user:** all
**expected behaviour:** notification is sent once
**behaviour:** email is sent twice at the same time to the same participant
**important screensize:**
**device & browser:** 
**Comment/Question:** tested with a user test account. Email was sent to the same email ****+user@liqd.net
Question: How often and when is the reminder sent? A user on a+ prod reported that reminder emails are sent out more often than once. Thus, has been there before django update

![Bildschirmfoto 2022-01-18 um 11 12 10](https://user-images.githubusercontent.com/36957474/149917024-22ed3971-3e80-466a-a2fd-afc70a4dc577.png)

**Comment by fuzzylogic2000** [comment]
I did notice that before, but always thought, that there were simply two ending phases... 
And I have questions:
1. is this also the case for the phase start notifications?
2. @CarolingerSeilchenspringer Is this the same in mB?

**Comment by fuzzylogic2000** [comment]
What I found out so far: there are also two actions for the (three??) tasks on dev:
```
>>> Action.objects.filter(project=246).first().timestamp
datetime.datetime(2022, 1, 17, 17, 23, 5, 413372, tzinfo=<UTC>)
>>> Action.objects.filter(project=246).first().id
1435
>>> Action.objects.filter(project=246).last().timestamp
datetime.datetime(2022, 1, 17, 17, 23, 5, 327802, tzinfo=<UTC>)
>>> Action.objects.filter(project=246).last().id
1434
```

**Comment by lthueer** [comment]
@fuzzylogic2000 on a+ dev emails are only sent twice (or even more, for e.g. to moderators) when the participation is about to end. For the start notification it's not, it only comes once!

**Comment by CarolingerSeilchenspringer** [comment]
@fuzzylogic2000 no, all fine on mB, tested both start and end. As it should (although not optimal) you only get mails twice for multimodule projects where phases end at the same time.

**Comment by fuzzylogic2000** [comment]
I did have a long look and didn't come closer to solving it:
- it only happens from time to time (checked django-admin for completed tasks with`NotifyFollowersOnPhaseIsOverSoonEmail` and when there were more than one, I also checked the adhocracy4.actions.model.Action by the Action number giving in the tasks).
- as the action also exists twice for the double sent emails, the problem comes from there. But it is weird, that that should only be in a+ and not mB as they use the same code from a4 (https://github.com/liqd/adhocracy4/blob/803c4c3b0ae83007cdc7c1971348e78abe92a3d1/adhocracy4/actions/management/commands/create_system_actions.py#L66). mB and a+ have the same setting (48 hours) for 

Another example:
